__tests__/DMT.test.js

```diff
@@ -225,23 +192,21 @@ describe('Test suite for device mode transformation feature', () => {
     xhrMockBadReq.onreadystatechange();
   };
 
+  const mockCreatePayload = jest.fn(() => createPayloadResponse);
+
   beforeEach(() => {
-    payload = TransformationsHandler.createPayload(event);
+    payload = mockCreatePayload(event);
```

Comment:
Why do you need a mock here?

Reply:
Misunderstood your suggestion. Now Addressed.